fastapi/routing.py

```diff
@@ -125,7 +125,12 @@ async def serialize_response(
             response_content,
             exclude_unset=exclude_unset,
             exclude_defaults=exclude_defaults,
-            exclude_none=exclude_none,
+            # jsonable_encoder() below will apply exclude_none, so it is not
+            # necessary here. Instead excluding None here may even cause issues
+            # as not all fields with a value of None have a None default value
+            # and by that would be initialized with None after the validation
+            # phase again.
+            exclude_none=False,
```

Comment:
I Think this comment is unnecessary. I would just remove the line with `exclude_none=`